src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -5284,6 +5321,32 @@ internal bool CheckValEscape(SyntaxNode node, BoundExpression expr, SafeContext
 
                 case BoundKind.IncrementOperator:
                     var increment = (BoundIncrementOperator)expr;
+                    if (increment.MethodOpt is { IsStatic: true } incrementMethod)
+                    {
+                        var prefix = increment.OperatorKind is UnaryOperatorKind.UserDefinedPrefixIncrement or UnaryOperatorKind.UserDefinedPrefixDecrement;
+                        Debug.Assert(prefix || increment.OperatorKind is UnaryOperatorKind.UserDefinedPostfixIncrement or UnaryOperatorKind.UserDefinedPostfixDecrement);
+
+                        // Prefix increment can be analyzed like the underlying method call since that's what it returns.
+                        // Postfix increment is better analyzed as only the operand since that's what it returns.
+                        if (prefix)
+                        {
+                            return CheckInvocationEscape(
```

Comment:
>CheckInvocationEscape [](http://example.com/codeflow?start=35&length=21)

Is this code path covered by a test?

Reply:
Yes, `RefEscapingTests.UserDefinedIncrementOperator_RefStruct_Scoped_Prefix`.